homeworks/taras.bobov_LotusElize/GamesProject/success.html

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <!-- <link rel="stylesheet" href="css/style.css"> -->
+        <!-- <link rel="stylesheet" href="css/styles.css"> -->
```

Comment:
Ouch, why do you like to edit files of another student?

Reply:
@vladyslav-yermolin-moc My fault. Sorry. Auto-refactor plus my inattention :(